pairing.php

```diff
@@ -45,8 +47,28 @@ function calculate(PDO $pdo): self
     }
 
     //TODO: generate sql to add the pairing to completed rounds
-    function sql($id, $i1, $i2): string
+    function sql(PDO $pdo, string $id, item $i1, item $i2): string
```

Comment:
We probably do not need to put item1 and item2 in the parameters, we can just grab them from the object